### PR TITLE
Rename ovmpt2x , ovmpt2ga and ovmpt2a

### PR DESCRIPTION
**Pull request by jkingdon**
Rename ovmpt2x to ovmpox

Rename ovmpt2ga to ovmpoga

Rename ovmpt2a to ovmpoa

This is in set.mm and iset.mm

### CODE REVIEW

File: iset.mm [hunk 1]
```diff
@@ -56423,7 +56423,7 @@ associative structure (such as a group).  (Contributed by NM,
   the second argument is the first component of the first argument, in short
   "x-maps-to operations".  For labels, the abbreviations "mpt2x" are used
   (since "x" usually denotes the first argument).  This is in line with the
-  currently used conventions for such cases (see ~ cbvmpox , ~ ovmpt2x and
+  currently used conventions for such cases (see ~ cbvmpox , ~ ovmpox and
   ~ fmpt2x ).  However, we are in the process of renaming to use the
   abbreviation "mpo" instead of "mpt2". As this proposal is realized, the
   labels in the following should also be adapted. If the first argument is
```
Comment (avekens) [inline iset.mm:56429]: Please don#t forget to revise this comment after finishing the replacements.

File: set.mm [hunk 2]
```diff
@@ -74306,7 +74306,7 @@ any sets (which usually are functions) and any element (even not
   the second argument is the first component of the first argument, in short
   "x-maps-to operations".  For labels, the abbreviations "mpt2x" are used
   (since "x" usually denotes the first argument).  This is in line with the
-  currently used conventions for such cases (see ~ cbvmpox , ~ ovmpt2x and
+  currently used conventions for such cases (see ~ cbvmpox , ~ ovmpox and
   ~ fmpt2x ).  However, we are in the process of renaming to use the
   abbreviation "mpo" instead of "mpt2" (see ~ conventions-labels ). As this
```
Comment (avekens) [inline set.mm:74311]: See remark on the corresponding comment in iset.mm

**Review by avekens — APPROVED** [review]

File: iset.mm (hunk 1 above)
Comment (jkingdon) [reply iset.mm:56429]: Good point, I'll add a checklist in the GitHub issue

Comment (jkingdon) [reply iset.mm:56429]: I have added the checklist to the top of https://github.com/metamath/set.mm/issues/98 . Please edit that issue or let me know if there is anything I missed.

**Pull request merged**